pkg/rtcmos/audioscore.go

```diff
@@ -38,12 +40,13 @@ func AudioScore(input Stat) Scores {
 	}
 
 	Ipl := Ie + (100-Ie)*(pl/(pl+Bpl))
+
 	delayFactor := float64(0)
 	if delay > 150 {
-		delayFactor = 0.1 * delay
+		delayFactor = 0.1 * (delay - 150)
```

Comment:
Fixing a bug here. Actually, there are two bugs
1. This part of code was not subtracting the 150 like here - https://github.com/ggarber/rtcscore/blob/9bd2b6d5ad32d59cd650383b9e0147a87dac3dbe/src/rtc_mos.js#L37
2. But, I think that code in Gustavo's repo itself has a mistake with operator precedence. It should be multiplying 0.1 to delay over 150. A paper reference for that - https://www.hamilton.ie/dwmalone/assem_et_al.pdf (Equation 9)